public/index.php

```diff
@@ -62,7 +62,7 @@
 $routes = require __DIR__ . '/../app/routes.php';
 $routes($app);
 
-if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'production') {
+if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'prod') {
```

Comment:
@okiedork should this be kept as "production"?

Reply:
`prod`.